scripts/shell_provisioner.sh

```diff
@@ -53,7 +53,7 @@ elif [ "$OSTYPE" = "Ubuntu" ]; then
         rm -f /tmp/puppetlabs.deb
         apt-get update
     fi
-elif [ "$OSTYPE" = "RedHat" ]; then
+elif [ "$OSTYPE" = "RedHat" ]  || [ "$OSTYPE" = "CentOS" ]; then
```

Comment:
In all my centos servers (physical, virtual, 6, 7) it looks like this:

<pre>
[cloud-user@icinga2 ~]$ lsb_release -i -s
CentOS
</pre>

Apparently in some cases it does turn into CentOS